packages/clay-navigation-bar/docs/index.js

```diff
@@ -39,51 +38,88 @@ render(<Component />);`;
 
 const NavigationBar = () => {
 	const scope = {ClayLink, ClayNavigationBar};
+
 	const code = NavigationBarCode;
 
 	return (
 		<Editor code={code} imports={navigationBarImportsCode} scope={scope} />
 	);
 };
 
-const navigationBarWithStyledItemImportsCode = `import ClayNavigationBar from '@clayui/navigation-bar';
-`;
+const navigationBarWithStyledItemImportsCode = `import ClayNavigationBar from '@clayui/navigation-bar';`;
 
 const NavigationBarWithStyledItemCode = `const Component = () => {
 
-    const btnStyle = {
-        padding: "5.5px 16px 5.5px 16px",
-        borderColor: "var(--indigo)"
-    };
-
-    return (
-        <ClayNavigationBar triggerLabel="Item 1" spritemap={spritemap}>
-            <ClayNavigationBar.Item active>
-                    <button className="btn btn-unstyled btn-block btn-sm" style={btnStyle} type="button">Item 1</button>
-            </ClayNavigationBar.Item>
-            <ClayNavigationBar.Item>
-                    <button className="btn btn-unstyled btn-block btn-sm" style={btnStyle} type="button">Item 2</button>
-            </ClayNavigationBar.Item>
-            <ClayNavigationBar.Item>
-                    <button className="btn btn-unstyled btn-block btn-sm" style={btnStyle} type="button">Item 3</button>
-            </ClayNavigationBar.Item>
-        </ClayNavigationBar>
-    );
+	const btnStyle = {
+		padding: "5.5px 16px 5.5px 16px",
+		borderColor: "var(--indigo)"
+	};
+
+	return (
+		<ClayNavigationBar triggerLabel="Item 1" spritemap={spritemap}>
+			<ClayNavigationBar.Item active>
+					<button className="btn btn-unstyled btn-block btn-sm" style={btnStyle} type="button">Item 1</button>
+			</ClayNavigationBar.Item>
+			<ClayNavigationBar.Item>
+					<button className="btn btn-unstyled btn-block btn-sm" style={btnStyle} type="button">Item 2</button>
+			</ClayNavigationBar.Item>
+			<ClayNavigationBar.Item>
+					<button className="btn btn-unstyled btn-block btn-sm" style={btnStyle} type="button">Item 3</button>
+			</ClayNavigationBar.Item>
+		</ClayNavigationBar>
+	);
 }
-    
+	
 render(<Component />);`;
 
+const navigationBarJSPImportsCode = `<%@ taglib uri="http://liferay.com/tld/clay" prefix="clay" %>`;
+
+const NavigationBarJSPCode = `<clay:navigation-bar
+	navigationItems='<%=
+		new JSPNavigationItemList(pageContext) {
+			{
+				add(
+					navigationItem -> {
+						navigationItem.setActive(true);
+						navigationItem.setHref(StringPool.BLANK);
+						navigationItem.setLabel("Item 1");
+					}
+				);
+				add(
+					navigationItem -> {
+						navigationItem.setHref(StringPool.BLANK);
```

Comment:
We don't need this no-more. We were forced before by `Soy` since it was a mandatory field, but it generates the wrong markup, so better remove the empty `setHref`.